connect/api.go

```diff
@@ -87,6 +87,36 @@ func upgradeProduct(product Product) (Service, error) {
 	return remoteService, err
 }
 
+func activateProduct(product Product, email string) (Service, error) {
+	var payload = struct {
+		Indentifier string `json:"identifier"`
+		Version     string `json:"version"`
+		Arch        string `json:"arch"`
+		ReleaseType string `json:"release_type"`
+		Token       string `json:"token"`
+		Email       string `json:"email"`
```

Comment:
email is optional according to the docs

Reply:
I don't think it matters. The Ruby version sends `"email": null`

Reply:
Ok